src/components/loginComponents/InputForm.tsx

```
@@ -0,0 +1,49 @@
+import { UseFormRegister } from 'react-hook-form';
+import Text from '../common/Text';
+
+interface InputFormProps {
+  label: string;
+  type: string;
+  name: string;
+  placeholder: string;
+  required: boolean;
+  register: UseFormRegister<any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)